Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -876,14 +1019,388 @@ def fetch_incidents(client, last_run, first_fetch_time, min_severity):
 
     next_run = {
         'last_fetch_time': latest_created_time.strftime(DATE_FORMAT),
-        'last_fetch_ids': current_fetch_ids
+        'last_fetch_ids': current_fetch_ids,
     }
     return next_run, incidents
 
 
+def threat_indicators_data_to_xsoar_format(ind_data):
+    """
+    Convert the threat indicators data from the raw to XSOAR format.
+
+    :param ind_data: (dict) The incident raw data.
+    """
+
+    properties = ind_data.get('properties', {})
+
+    formatted_data = {
+        'ID': ind_data.get('id'),
+        'Name': ind_data.get('name'),
+        'ETag': ind_data.get('etag'),
+        'Type': ind_data.get('type'),
+        'Kind': ind_data.get('kind'),
+
+        'Confidence': properties.get('confidence', ''),
+        'Created': properties.get('created', ''),
+        'CreatedByRef': properties.get('createdByRef', ''),
+        'ExternalId': properties.get('externalId', ''),
+        'LastUpdatedTimeUtc': properties.get('lastUpdatedTimeUtc', ''),
+        'Revoked': properties.get('revoked', ''),
+        'Source': properties.get('source', ''),
+        'ThreatIntelligenceTags': properties.get('threatIntelligenceTags', ''),
+        'Tags': properties.get('threatIntelligenceTags', 'No Tags'),
+        'DisplayName': properties.get('displayName', ''),
+        'Description': properties.get('description', ''),
+        'Types': properties.get('threatTypes', ''),
+        'KillChainPhases': [{
+            "KillChainName": phase.get("killChainName"),
+            "PatternTypeValues": phase.get("phaseName")
+        }
+            for phase in properties.get('KillChainPhases', [])],
+
+        "ParsedPattern": [{
+            "patternTypeKey": pattern.get("patternTypeKey"),
+            "patternTypeValues": pattern.get("patternTypeValues")
+        }
+            for pattern in properties.get("parsedPattern", [])],
+
+        'Pattern': properties.get('pattern', ''),
+        "PatternType": properties.get("patternType", ''),
```

Comment:
not an error, but a convention: stick to either `'` or `"` to wrap strings, but avoid doing both.